api/framer-motion.api.md

```diff
@@ -224,9 +224,9 @@ export interface DraggableProps extends DragHandlers {
 // @public (undocumented)
 export interface DragHandlers {
     onDirectionLock?(axis: "x" | "y"): void;
-    onDrag?(event: MouseEvent | TouchEvent | PointerEvent, info: PanInfo): void;

```

Comment:
This IMO isn't a major API change as we used `PointerEvent`s as first choice and all major browsers support this.